DotNetHowToAutocomplete/README.md

```diff
@@ -0,0 +1,2 @@
+# AutocompleteTutorial
+An effective search experience is central to user productivity in most data centric applications whether it’s a web site or mobile app.  Users need to find what they are looking for as quickly and easily as possible.  One technique for improving productivity is bringing the richness of search directly to the user as they type.  In this tutorial we’ll show how suggestions, autocomplete, and facets in Azure Search can help you build a powerful search box.
```

Comment:
Please add this to the `README.md` file in the root of the repo.

Reply:
Will do.